core/src/test/scala/org/apache/pekko/persistence/r2dbc/internal/MonoToFutureSpec.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+package org.apache.pekko.persistence.r2dbc.internal
+
+import org.scalatest.TestSuite
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import reactor.core.publisher.Mono
+
+class MonoToFutureSpec extends AnyWordSpec with ScalaFutures with TestSuite with Matchers {
+  "MonoToFutureSpec" should {
+    "convert a Mono to a Future" in {
+      val r = Mono.just("pekko")
```

Comment:
hard string.
Adding test for empty and seq stream should be better . I am not very similar with Mono.

Reply:
Mono is just some kind of Future, which can only emit one/zero elements.